lib/Bio/EnsEMBL/DataCheck/Checks/SourceAdvisory.pm

```diff
@@ -0,0 +1,71 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::SourceAdvisory;
```

Comment:
Having 'Advisory' in the name doesn't feel quite right, but I don't have a better suggestion. So this is fine, but if anything occurs to me I'll propose a change at a later date...

Reply:
I agree 'Advisory' is not a good idea but as it's checking different columns and values, I couldn't find a better description.